README.md

```diff
@@ -0,0 +1,53 @@
+React-Native-Template
+================================================
+The goal of this project is to work as template for react-native applications, providing a base project structure, core dependencies and boilerplate to jumpstart development.
+
+## Base dependencies
+  - [axios](https://github.com/axios/axios) for networking.
+  - [PropTypes](https://github.com/facebook/prop-types) to type-check our components exposed propperties.
+  - [React-Native-Config](https://github.com/luggit/react-native-config) to manage envionments.
+  - [React-Native-Navigation](https://wix.github.io/react-native-navigation/#/) native navigation library.
+  - [Redux](https://redux.js.org/) for state management.
+  - [Recompose](https://github.com/acdlite/recompose) for utilities.
+  - [Redux-Persist](https://github.com/rt2zz/redux-persist) as persistance layer.
+  - [Redux-Thunk](https://github.com/gaearon/redux-thunk) to dispatch asynchronous actions.
+  - [Reselect](https://github.com/reactjs/reselect) the selector library for redux.
+  - [Jest](https://facebook.github.io/jest/) and [Enzyme](https://github.com/airbnb/enzyme) for testing.
+
+## Usage
+
+### Option #1: Use React-Native-Rename
+You can start by cloning this repository and using [react-native-rename](https://github.com/junedomingo/react-native-rename). In the current state of this project, it should give you no issues at all, just run the script, delete your node modules and reinstall them and you should be good to go.
+
+Keep in mind that this library can cause trouble if you are renaming a project that uses `Pods` on the iOS side.
+
+### Option #2: Copy the structure to your project
+If you want to roll on your own and don't want to use this as a template, you can create your own project and then copy the `/src` folder (which has all the code of your application) and update your `index.js`.
+
+Keep in mind that if you do this, you'll have to install _and link_ all dependencies (native, and pure-JS).
```

Comment:
maybe we can also specify that besides installing and linking, the react-native-navigation native code setup should be done.

Reply:
Yeah, I know, but maybe some people would think it's just running `react-native link` or something.